server/data/quotes.json

```diff
@@ -447,6 +447,14 @@
     "author": "Safir Kassim Boudjelal",
     "sentence": "You want the 3 golden rules to learn any language? Use it or lose it. Love it or leave it. Feel it or fail it."
   },
+  {
+    "author": "Safir Kassim Boudjelal",
+    "sentence": "Good writers are good readers and listeners."
```

Comment:
This quote is not related to "learning a foreign language" or something about language. Please change it. It doesn't meet the [requirements](https://github.com/jayehernandez/letra-extension/blob/master/docs/new_quote.md#-checklist).